layout-and-navigation/lesson-1.md

```diff
@@ -42,7 +42,7 @@ app = new kendo.mobile.Application(document.body, { layout: "main-layout" });
 ```
 
 * **f**. Save the app.js file.
-* **g**. Open the iPhone simulator (select **Run** --> **iPhone**) to see your new layout in action. Please note the defined layout will appear specific to the particular platform. 
+* **g**. Open the iPhone simulator (select **Run** --> **Open Simulator**) and then select any iPhone from the **iOS Devices** section in the drop-down menu located in the top-left corner to see your new layout in action. Please note the defined layout will appear specific to the particular platform. 
```

Comment:
I would suggest re-arranging the sentence:
 To see your new layout in action, select **Run** --> **Open Simulator** and then select any iPhone from the **iOS Devices** section in the drop-down menu located in the top-left corner